internal/provider/resource_group_member.go

```diff
@@ -147,7 +147,9 @@ func resourceGroupMemberCreate(ctx context.Context, d *schema.ResourceData, meta
 	}
 
 	member, err := membersService.Insert(groupId, &memberObj).Do()
-	if err != nil {
+
+	// If we receive a 409 that the member already exists, ignore it, we'll import it next
+	if err != nil && !memberExistsError(err) {
```

Comment:
kinda cool to essentially have create be import.. but I dunno if this is an accepted pattern? Would we not want things to error to sort of nudge the user towards "hey if this already exists, do an import instead"

Reply:
Do we have any other precedent for this pattern?

Reply:
Seems like its to get around eventual consistency problems, which makes sense then.